api/WebApi.ts

```diff
@@ -136,16 +138,24 @@ export class WebApi {
         }
 
         let userAgent: string;
-        const nodeApiName: string = 'azure-devops-node-api';
-        const nodeApiVersion: string = JSON.parse(fs.readFileSync(path.resolve(__dirname, 'package.json'), 'utf8')).version;
-        const osName: string = os.platform();
-        const osVersion: string = os.release();
-
-        if (requestSettings) {
-            userAgent = `${requestSettings.productName}/${requestSettings.productVersion} (${nodeApiName} ${nodeApiVersion}; ${osName} ${osVersion})`;
-        }
-        else {
-            userAgent = `${nodeApiName}/${nodeApiVersion} (${osName} ${osVersion})`;
+        if(isBrowser) {
+            if(requestSettings) {
+                userAgent = `${requestSettings.productName}/${requestSettings.productVersion} (${window.navigator.userAgent})`
```

Comment:
Can we still include nodeApiName here? aka `${requestSettings.productName}/${requestSettings.productVersion} (${nodeApiName}; ${window.navigator.userAgent})`